apps/web/components/booking/pages/AvailabilityPage.tsx

```diff
@@ -23,8 +23,9 @@ import notEmpty from "@calcom/lib/notEmpty";
 import { getRecurringFreq } from "@calcom/lib/recurringStrings";
```

Comment:
We need to handle the case where an event type turns the event type into an event without seats, but have existing seat bookings.